src/main/java/io/jenkins/plugins/neuvector/NeuVectorWorker.java

```diff
@@ -577,6 +583,36 @@ private void getToken(CloseableHttpClient httpclient) throws AbortException {
                 }
                 throw new AbortException(causeMessage);
             }
+        }  catch (SSLHandshakeException e) {
+            String causeMessage = "Server certificate verification error: " + e.getCause().getCause().getMessage();
```

Comment:
It might not be very safe to call e.getCause().getCause(), try something like this:
  if ((e.getCause() instanceof CertificateException
    && !(e.getCause().getMessage()...